packages/vulcan-lib/lib/modules/fragmentTypes.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This is in `vulcan-lib/lib/modules`, but the config autogenerates a file which looks like this in `vulcan-lib/lib` (without the modules). This should be moved up a level (if we're checking in this generated file), or deleted (if we aren't).